cinema/serializers.py

```diff
@@ -43,6 +53,7 @@
     actors = serializers.SlugRelatedField(
         many=True, read_only=True, slug_field="full_name"
     )
+    image = MovieImageSerializer
```

Comment:
The `image` field in `MovieListSerializer` is incorrectly assigned to `MovieImageSerializer`. Consider using a `SerializerMethodField` or similar to handle nested serialization properly.